ExcelWakaTimeAddin/Properties/AssemblyInfo.cs

```diff
@@ -33,6 +33,6 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("2.1.1")]
-[assembly: AssemblyFileVersion("2.1.1")]
+[assembly: AssemblyVersion("4.2.1")]

```

Comment:
We don't want to bump the extension version. This is different from the `WakaTime.Shared.ExtensionUtils` version.